interfaces/builtin/pluggable_storage.go

```diff
@@ -0,0 +1,65 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2016 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+import (
+	"github.com/snapcore/snapd/interfaces"
+)
+
+const pluggableStorageConnectedPlugAppArmor = `
+# Description: Can mount and unmount removable storage. This is restricted
+# because it gives privileged access to mount commands and should only be used
+# with trusted apps.
+# Usage: reserved
+
+# Needed for mount/unmount operations
+capability sys_admin,
+
+# TODO: These should be confined by path, need to come up with reasonable
+# values
+mount,
```

Comment:
Lets make this more finegrained (see http://wiki.apparmor.net/index.php/AppArmor_Core_Policy_Reference#Mount_rules_.28AppArmor_2.8_and_later.29) so that the user of this can only mount to SNAP_DATA or SNAP_USER_DATA and only specific devices.

mount /dev/sd* -> /var/snap/${SNAP_NAME}/${SNAP_REVISION}/**
umount /dev/sd* -> /var/snap/${SNAP_NAME}/${SNAP_REVISION}/**